Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift

```diff
@@ -14,11 +14,43 @@
 import SwiftUI
 import ArcGIS
 
-/// The `BasemapGallery` tool displays a collection of basemaps from either
-/// ArcGIS Online, a user-defined portal, or an array of `BasemapGalleryItem`s.
-/// When a new basemap is selected from the `BasemapGallery` and the optional
-/// `BasemapGalleryViewModel.geoModel` property is set, then the basemap of the
-/// `geoModel` is replaced with the basemap in the gallery.
+/// The `BasemapGallery` displays a collection of basemaps from ArcGIS Online, a user-defined
+/// portal, or an array of `BasemapGalleryItem`s. When a new basemap is selected from the
+/// `BasemapGallery` and the optional `BasemapGalleryViewModel.geoModel` property is set, the
+/// basemap of the `geoModel` is replaced with the basemap in the gallery.
+///
+/// | iPhone | iPad |
```

Comment:
Unfortunately here and other similar locations, the table headers are rendering in quick help. I haven't been able to find a way around it.

<img width="266" alt="badMD" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/d4a4a166-60e5-4a70-add1-7433dfe893a9">

Reply:
Could this go in an [extension file](https://developer.apple.com/documentation/xcode/adding-supplemental-content-to-a-documentation-catalog#Add-extension-files-to-append-to-or-override-source-documentation-comments)?

Reply:
Possibly so. I did just find that this behavior is fixed as of Xcode 15 (and looks quite nice) so it might not be that big a deal after-all.

<img width="50%" alt="Screenshot 2023-08-10 at 16 07 18" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/cb48b524-c03c-4a35-91e6-a1e3d5c0dbc1">

Reply:
I also experimented around some with the extension files and found two issues:

1. We loose control of where the extended content is placed. It either fully replaces the inline doc or appends to the end of it. So if we wanted to retain control of where the images are placed, we'd need to put all of the content into the extension.
2. In Xcode 14, even images in extended content don't render in Quick Help.

<img width="524" alt="Screenshot 2023-08-13 at 16 05 45" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/2959f6aa-a543-4122-9d1c-40c0e5992adf">

Reply:
@philium, Mark and I discussed this some more yesterday and our thinking is given that Xcode 15 will probably have a full release within a month or so and that the tables render okay in 15's Quick Help, we'll proceed with this as is.

Reply:
From the [DocC doc](https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files#Create-a-richer-experience-for-your-symbol-documentation):

> - Use extension files to provide additional content for your symbols, such as code examples and images, and **to help keep the size of their in-source comments manageable**. [emphasis mine]